static-services.tf

```diff
@@ -87,10 +122,97 @@ SECONDARY_SECRET=${var.stateless_dns_secondary_key}
+  providers = {
+    aws = "aws.us-west-2"
+  }
+  env_vars = <<EOF
+PORT=12345
+JWT_SECRET_KEY=${var.statsum_jwt_secret_key}
+SENTRY_DSN=${var.statsum_sentry_dsn}
+SIGNALFX_TOKEN=${var.statsum_signalfx_token}
+EOF
+}
+
+module "webhooktunnel" {
+  source                   = "modules/static-service"
+  log_host                 = "${var.static_service_log_host}"
+  log_port                 = "${var.static_service_log_port}"
+  security_groups          = ["${aws_security_group.basic_https.id}"]
+  nametag                  = "Webhook Tunnel"
+  servicetag               = "webhooktunnel"
+  instance_type            = "t2.medium"
+  runtime_name             = "webhooktunnel"
+  runtime_description      = "tunnel for webhooks"
+  runtime_port_map         = "443:12345"
+  image_tag                = "taskcluster/webhooktunnel"
+  image_hash               = "ed9a8c9f3949f3c8251d1d9a1ae3cfb0aaa99d584b707d12acfdf60f356cc3d5"
+  providers = {
+    aws = "aws.us-west-2"
+  }
+  env_vars = <<EOF
+PORT=12345
+ENV='production'
+HOSTNAME=${var.webhooktunnel_hostname}
+TASKCLUSTER_PROXY_SECRET_A=${var.webhooktunnel_secret_a}
+TASKCLUSTER_PROXY_SECRET_B=${var.webhooktunnel_secret_b}
+TLS_KEY='HELP WHAT IS THE PLAN HERE'
+TLS_CERTIFICATE='HELP WHAT IS THE PLAN HERE'
+EOF
+}
+
+module "cloud-mirror-copiers" {
+  source                      = "modules/static-service"
+  log_host                    = "${var.static_service_log_host}"
+  log_port                    = "${var.static_service_log_port}"
+  security_groups             = ["${aws_security_group.deny_all.id}"]
+  instances                   = 2
+  service_copies_per_instance = 3
+  nametag                     = "Cloud-Mirror Copiers"
+  servicetag                  = "cloud-mirror-copiers"
+  instance_type               = "c4.8xlarge"
+  runtime_name                = "cloudmirrorcopiers"
+  runtime_description         = "cloud-mirror processes to copy data from region to region"
+  image_tag                   = "taskcluster/cloud-mirror"
+  image_hash                  = "f9aa7c009de17504da08e4725a3c6cfb1a3785bdd2cfdb5f05dc53ac8fa54182"
+  providers = {
+    aws = "aws.us-west-2"
+  }
+  env_vars = <<EOF
+AWS_ACCESS_KEY_ID='...'
+AWS_SECRET_ACCESS_KEY='...'
+DEBUG='* -aws -babel -base:validator -eslint* -express:* -sqs-consumer -superagent -base:stats -cloud-mirror:aws-* -taskcluster-lib-monitor'
+NODE_ENV='production'
+PULSE_PASSWORD='...'
+PULSE_USERNAME='...'
+REDIS_HOST='...'
+REDIS_PASS='...'
+REDIS_PORT='...'
+TASKCLUSTER_ACCESS_TOKEN='...'
+TASKCLUSTER_CLIENT_ID='...'
+EOF
+}
+
+output stateless_dns_ips {
+  value = "${concat(module.stateless_dns_us_west_2.static_ips, module.stateless_dns_eu_west_1.static_ips)}"
+}
+
+output statsum_ips {
+  value = "${module.statsum.static_ips}"
 }
 
-output stateless_dns_eu_west_1_ip {
-  value = "${module.stateless_dns_eu_west_1.static_ip}"
+output webhooktunnel_ips {
+  value = "${module.webhooktunnel.static_ips}"
```

Comment:
What is this... maybe a doc comment? please...

Also what is it used for, I'm curious...

Reply:
This is just the ip addr that we can point at to access the service.